build/build-image/cross/Dockerfile

```diff
@@ -62,14 +62,13 @@ RUN echo "deb http://archive.ubuntu.com/ubuntu xenial main universe" > /etc/apt/
 
 # work around 64MB tmpfs size in Docker 1.6
 ENV TMPDIR /tmp.k8s
-
-# Get the code coverage tool, goimports, and godep
 RUN mkdir $TMPDIR \
   && chmod a+rwx $TMPDIR \
-  && chmod o+t $TMPDIR \
-  && go get golang.org/x/tools/cmd/cover \
-            golang.org/x/tools/cmd/goimports \
-            github.com/tools/godep
+  && chmod o+t $TMPDIR
+
+# Get the code coverage tool and goimports
```

Comment:
@ixdy Do we still need these? It would be great if we didn't have to `go get` them.